### ISSUE TITLE
MultiSelect: scrolling using arrow keys does not work with VirtualScroll

### ISSUE THREAD

**Issue by donSan4o** [primefaces/primereact#7478]
### Describe the bug

Scroll doesn't work when trying to choose some option with keyboard arrow down or up keys in virtual scroll case for example

### Reproducer

https://primereact.org/multiselect/#virtualscroll

### System Information

```Shell
primereact 10.8.5
```

### Expected behavior

scroll works as expected with keyboard
